scripts/ock-cleanup.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+########### generic ###########
+title()
+{
+    case "$1" in
+    "SECTION")
+        shift 1
+        echo "########################################"
+        echo "## $*"
+        echo ""
+        ;;
+    "ENDSECTION")
+        echo ""
+        echo "                                      ##"
+        echo "########################################"
+        echo ""
+        ;;
+    "PARA")
+        shift 1
+        echo ""
+        echo "## $*"
+        ;;
+    "LINE")
+        shift 1
+        echo "$*"
+        ;;
+    *)
+        echo "$*"
+        ;;
+    esac
+}
+
+########### ock-specific ###########
+
+SLOT=${OCK_SLOT:-"3"}
+PIN=${OCK_PIN:-"12345678"}
+
+TMPDIR="tmp-ock"
```

Comment:
TMPDIR is used by some user applications, please use TMPPDIR (see the change I made to the current tests recently)

Reply:
ok, I will check this.